droidlet/lowlevel/minecraft/tests/test_cuberite_agent_connection.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+import os
+import signal
+import subprocess
+import time
+import unittest
+from agents.craftassist.craftassist_agent import CraftAssistAgent
+from agents.craftassist.tests.fake_agent import MockOpt
+
+
+class MyTestCase(unittest.TestCase):
+    def test_agent_cuberite_connection(self):
+        opts = MockOpt()
+        opts.port = 25565
+        cuberite_process_path = "droidlet/lowlevel/minecraft/cuberite_process.py"
```

Comment:
Nit : once we do the above lets use the standard `os.path.join` instead of hardcoding. So something like :
```
repo_home = os.path.dirname(os.path.realpath(__file__))
cuberite_process_path = os.path.join(repo_home, "../../droidlet/lowlevel/minecraft/cuberite_process.py")
```